PR #9 tweaks

secretlycarl opened the pull request:
Bunch of tweaks and some fixes for @mon5termatt big PR.

- fixed some bugs and tweaked some wording on setup

- fixed services still showing on admin page despite no associated URL entered on setup

- section 5 in onboarding now disappears if there is no pulsarr or overseer URL, and the sections after are renumbered

- added accent color picker and logo uploads to setup and admin page

- cleaned up media lists page

fixes needed-

- fix logo upload handling/conversion. makes transparent pngs and webps have black bg

- new accent color picker doesn't work on /services, works fine on /setup

- open browser on first startup works when running from terminal, not in docker

- faster/more efficient library poster/data loading. takes a long time with many libraries



after all of that is smoothed out I'm going to work on the CSS and layout a bit

mon5termatt commented:
The Accent color doesnt actually save the change when you select a color using the picker. it should write it to the box next to it after selecting a color.

secretlycarl commented:
fixed library names on admin panel, admin color picker, poster cache limit on main page, added user invite/creation on admin panel

Still needs 

- fix logo upload handling/conversion. makes transparent pngs and webps have black bg
